js102/tests-skip-some-tests-on-rv64.patch

```diff
@@ -0,0 +1,16 @@
+--- a/js/src/tests/non262/Array/regress-157652.js	2022-11-30 14:24:40.713125777 +0800
++++ b/js/src/tests/non262/Array/regress-157652.js	2022-11-30 16:46:55.001791795 +0800
+@@ -1,4 +1,4 @@
+-// |reftest| skip-if(xulRuntime.XPCOMABI.match(/x86_64|aarch64|ppc64|ppc64le|s390x|mips64|loongarch64/)||Android) -- No test results
++// |reftest| skip-if(xulRuntime.XPCOMABI.match(/x86_64|aarch64|ppc64|ppc64le|s390x|mips64||loongarch64|riscv64/)||Android) -- No test results
```

Comment:
There seems to be an additional "|" before loongarch64 here.

Reply:
fixed